### DISCUSSION TITLE
Billing in organization only shows total, not which repo

### DISCUSSION THREAD

**Discussion by mfn** [community/community#26150] [Actions]
<p>Hi,</p>
<p>is there a way to see what private repo consumed the most minutes for Github Action? The /organization/&lt;orgname&gt;/settings/billing only shows the used minutes for all private repos.</p>
<p>If I want to figure out which repo might consume the most, there’s no easy way currently. I can go manually through all repos (~10) but even there, there’s no “sum” so it’s still a guessing game. Especialy if a lot of workflows are active and lots of pushed are performed.</p>
<p>Is there a way to get more insights into this?</p>
<p>thanks,</p>
<ul>
<li>Markus</li>
</ul>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hello <a class="mention" href="/mfn">@mfn</a>!</p>
<p>Thanks for reaching out! We don’t currently have a way to view a per-repository breakdown of Actions usage, but it is something that our Actions and Billing teams are actively working on.  You may reach out to our <a href="https://support.github.com/" rel="nofollow noopener">private support</a> team to get more detail regarding current billing and usage for your account.</p>

**Comment by mfn** [comment]
<p>Hi,</p>
<p>thanks for the quick response, that’s great to hear! I think I can wait, no pressure at the moment.</p>
<p>Maybe I can make one more suggestion: besides an option to view a drill down on the usage, on the /actions tab itself it woul be great to (somewhere) show the “total consumed minute” of a workflow.</p>
<p>Currently the workflow time is the “wall time” how long basically the longest job (+ internal overhead) took, but if I was running 100 jobs each consuming 1 minute, the wall time is 1 minute but in fact I’m getting billed 100 minutes of course. This should be reflected somewhere too, because that is essentially what is being paid.</p>
<p>thanks!</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Thank you for the detail <a class="mention" href="/mfn">@mfn</a>! I’ve also added your feedback and example to the open internal feature request.</p>

**Comment by chrispat** [comment]
<p>You should soon see the ability to export a CSV of your billing details in your account or Org.  That export will enable you to report on the minutes used by specific repos and other dimenions.</p>

**Comment by ianwilliams1** [comment]
Referencing [my answer ](https://github.com/community/community/discussions/38770#discussioncomment-5199055) in a similar ticket.
